cfglib.py

```diff
@@ -141,7 +141,11 @@
     cfg.StrOpt('tasks_mapping', default='scenario/tasks.yaml',
                help='Path to a file which holds CloudFerry python code tasks '
                     'mapped to migration scenario items. Items defined in '
-                    'this file must be used in the migration scenario.')
+                    'this file must be used in the migration scenario.'),
+    cfg.BoolOpt('migrate_user_quotas', default=True,
```

Comment:
Since we have `migrate_user_quotas` option now, maybe it would be better to have smth like `migrate_tenant_quotas` (or `migrate_project_quotas`) option instead of just `migrate_quotas` as we currently already have. Because from my perspective it can make some misunderstandings...
 
Is it actually possible to make these 2 options separated and independent from each other fast and painlessly?